tests/test_agnostic/test_metadata_schema.py

```diff
@@ -155,6 +157,203 @@ def test_min_max(self):
             Axis(name="test", min=0, max=-10)
 
 
+class TestAffineTransformation:
```

Comment:
It looks like most of the tests here are duplicates of the tests in test_affine. Let's consolidate